custom_components/tuya_local/devices/blitzwoolf_bwsh5-humidifier.yaml

```diff
@@ -0,0 +1,193 @@
+# sample DPS {'updated_at': 1674438037.778799, '1': True, '4': 'health', '5': True, '10': 24, '12': 75, '13': 55, '14': 77, '18': 'c', '19': 'cancel', '20': 0, '21': True, '22': 0, '23': 'level_1', '26': True, '35': True} 
+name: BlitzWoolf BWSH5 Humidifier
+products: 
+  - id: lqzme7o0kzhumdxr
+primary_entity:
+  entity: humidifier
+  class: humidifier
+  dps:
+    - id: 1
+      name: switch   
+      type: boolean
+      mapping:
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          value: ON
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          value: OFF
+    - id: 5
+      name: led
+      type: boolean
+    - id: 10
```

Comment:
The attributes that have sensors defined below do not need to be added here as well, since they aren't a standard part of the humidifier entity interface so have no special meaning.